### PR TITLE
Claim BNT does not move over once countdown is done #632

### LINKED ISSUE #632: Claim BNT does not move over once countdown is done
**Expected behavior**
BNT positions to claim should move over to the left once time has passed. Refresh resolves it, but should not be necessary.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by sirius651**
Fixed #632 

스크린샷 2021-01-25 오후 12 21 22

Fixed current wrong implementation. After locking period, it should show off button to claim.

### CODE REVIEW

**Comment by Velua** [comment]
@pingustar please review.

File: src/components/protection/ClaimBnt.vue [hunk 1]
```diff
@@ -14,16 +14,13 @@
...
-        {{ `(~$${item.usdValue})` }}
-      </span> -->
+      <span class="amt-num mx-2">{{ `${prettifyNumber(item.amount)} BNT` }}</span>
     </div>
     <div v-if="!locked">
       <b-btn
         variant="primary"
         @click="click"
-        class="font-size-14 font-w500 px-4"
+        class="btn-claim font-size-14 font-w500 px-4"
```
Comment (pingustar) [inline src/components/protection/ClaimBnt.vue:23]: `btn-claim` class doesnt seem to exist

File: src/components/protection/ClaimBnt.vue [hunk 2]
```diff
@@ -14,16 +14,13 @@
         :src="bntLogoSrc"
         alt="Token Logo"
       />
-      <span class="mx-2">{{ `${prettifyNumber(item.amount)} BNT` }}</span>
-      <!-- <span class="text-primary font-size-12">
-        {{ `(~$${item.usdValue})` }}
-      </span> -->
+      <span class="amt-num mx-2">{{ `${prettifyNumber(item.amount)} BNT` }}</span>
```
Comment (pingustar) [inline src/components/protection/ClaimBnt.vue:17]: `amt-num` class doesnt seem to exist

**Review by pingustar — CHANGES_REQUESTED** [review]
@sirius651 just tested it and it does the same.

The claim button appears on the "Locked" side instead of moving over to the "Available to claim"

File: src/components/protection/ClaimBnt.vue (hunk 1 above)
Comment (sirius651) [reply src/components/protection/ClaimBnt.vue:23]: just set for testing, I'll try to find it without specific class name.

File: src/components/protection/ClaimBnt.vue (hunk 2 above)
Comment (sirius651) [reply src/components/protection/ClaimBnt.vue:17]: same above

**Comment by sirius651** [comment]
> @sirius651 just tested it and it does not move over automatically.
> 
> The claim button appears on the "Locked" side instead of moving over to the "Available to claim"

@pingustar yes, previously, it did not switch into claimable button after locking period, because still remain timer. so, I fixed it and It seemed clearer that the button changed at that location. do you think moving over to 'Available to Claim' would be more straightforward?

**Comment by pingustar** [comment]
@Velua do you know if thats okay to stay in the locked section? I remember something about claiming multiple locked BNT positions get merged and claimed at once.

Eg:

If you have 5 pending locked positions on the right side, they will get merged into one claimable action on the left side once the timer ended.

**Comment by Velua** [comment]
> @Velua do you know if thats okay to stay in the locked section? I remember something about claiming multiple locked BNT positions get merged and claimed at once.
> 
> Eg:
> 
> If you have 5 pending locked positions on the right side, they will get merged into one claimable action on the left side once the timer ended.

Correct, they should get consolidated, that way the user doesn't have to attempt to press it several times.

**Review by pingustar — CHANGES_REQUESTED** [review]
@sirius651 is this still a draft?

using the preview link I dont see any positions on locked nor on claimable.

Preview:
image

Master branch:
image

**Comment by sirius651** [comment]
> @sirius651 is this still a draft?
> 
> using the preview link I dont see any positions on locked nor on claimable.
> 
@pingustar  : once time over, the balance function was modified to recall. how can I reveal locked balance in Closed Positions? I can't see it from my side from some point on.

File: src/components/protection/AddProtectionSingle.vue [hunk 3]
```diff
@@ -296,7 +296,8 @@ export default class AddProtectionSingle extends BaseComponent {
   }
 
   get whitelistWarning() {
-    const msg = i18n.t("pool_not_approved");
+    const msg =
```
Comment (Velua) [inline src/components/protection/AddProtectionSingle.vue]: Are we meant to be losing the ```i18n.t``` feature?

Comment (sirius651) [reply src/components/protection/AddProtectionSingle.vue]: no, current on master branch, it use `i18n.t`. just following on master branch to merge

**Review by pingustar — APPROVED** [review]
Code looks good. Will test on ropsten.

**Pull request merged**